src/commonmark/en/content/developer/web-api/data.md

```diff
@@ -981,7 +981,13 @@ data values will behave just like other data values which store text
 content, but should be handled differently in order to produce
 meaningful input and output.
 
-The process of storing one of these data values roughly goes like this:
+There are two methods of storing FileResource data values.
+
+**The Easy Way:** Upload the file to the `/api/dataValues/file` endpoint as
```

Comment:
It's possible that someone will want to write frontend code that works on multiple versions of DHIS2. To facilitate this, I suggest something like:

The Easy Way (DHIS2 2.36 and later): ...
The Hard Way (compatible with earlier DHIS2 versions): ...

Reply:
What I wanted to convey is that the hard way could be used if you're developing for both 2.36 and earlier versions. But I don't need to insist on that.

Reply:
Tried again—does the revised text work?

I'm happy to try and clarify the commentary regarding versions, but it does feel like too much to include it in the bold text to me.